run.py

```diff
@@ -99,12 +99,14 @@ def get_iss_cmd(base_cmd, elf, log):
   return cmd
 
 
-def gen(test_list, simulator, simulator_yaml, output_dir, sim_only,
-        compile_only, lsf_cmd, seed, cwd, cmp_opts, sim_opts, timeout_s):
+def gen(test_list, csr_file, isa, simulator, simulator_yaml, output_dir, sim_only,
+        compile_only, lsf_cmd, seed, cwd, cmp_opts, sim_opts, iterations, timeout_s):
```

Comment:
Why not use test['iterations']?

Reply:
yes, the iterations argument is redundant since yaml processing has already happened